src/components/Header/Header.scss

```diff
@@ -0,0 +1,5 @@
+@import "@material/top-app-bar/mdc-top-app-bar";
+
+.mdc-drawer-app-content {
```

Comment:
please do not match/modify `material-components` classes - add your own BEM-styled class name and modify. Besides, you should wrap each markup into BEM component, or at least add BEM block name on top-level container